packages/oauth/src/index.ts

```diff
@@ -529,12 +538,16 @@ export class InstallProvider {
       }
 
       // Call the failure callback
+      const codedError: CodedError = error as CodedError;
+      if (codedError.code === undefined) {
```

Comment:
This never happens as long as bolt-js / `@slack/oauth` does not have internal bugs, though. This addition is mainly to satisfy the TS compiler.